dev/src-expose/entry.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env sh
+
+# Feed every directory in /app/data to src-expose
+codedirs=$(cd /app/data && ls -d * | xargs)
+/usr/local/bin/src-expose $codedirs
```

Comment:
if we have a non-zero exit code it won't be reported. Maybe do
```suggestion
exec /usr/local/bin/src-expose $codedirs
```